lambdas/get-user/tsconfig.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
not really part of this task. But there is a way inherit tsconfig from a parent file. 
Then we could include it here not have to copy it everytime.